packages/data-point-cache/lib/in-memory.js

```diff
@@ -20,27 +20,33 @@ function del (cache, key) {
   delete cache.entries[key]
 }
 
+/**
+ * NOTE: This method mutates the cache.entires array.
+ * @param {Object} cache - Object that holds a reference to list of cache entries
+ */
+function swipeTick (cache) {
+  const keys = Object.keys(cache.entries)
+  if (keys.length > 10000) {
+    cache.entries = {}
+    logger.warn(
+      'Cache inMemory reached max (1000) number of entries, all keys now being deleted.'
+    )
+    return
+  }
+  let entry
+  const now = Date.now()
+  for (let index = 0; index < keys.length; index++) {
+    let key = keys[index]
+    entry = cache.entries[key]
```

Comment:
I don't know if there's any pro/con, but the `entry` can be defined as a `const` inside the for loop.

Also, the `key` can be a `const`.